public/about.html

```diff
@@ -22,30 +48,48 @@ <h5> navPlace </h5>
             </p>
-            <a class="button primary" style="display: inline-block;" href="navPlace.html">
-                Create a navPlace Object
-            </a>
             <h5> Annotation </h5>
             <p>
                 The IIIF Presentation API incorporates the <a target="_blank" href="https://www.w3.org/TR/annotation-model/">W3C Web Annotation</a> for the "Annotation" objects created under its schema.  These Annotations can be embedded directly onto <a target="_blank" href="https://iiif.io/api/presentation/3.0/#21-defined-types">IIIF Defined Types</a>.  It can also target those data types as non-destructive, non-dispruptive independent data assertion.  This path seeks to finish with a real active <abbr title="Linked Open Usable Data">LOUD</abbr> Geolocating Web Annotation for the resource you provide.
             </p>
-            <a class="button primary" style="display: inline-block;" href="annotate.html">
-                Create a Geolocating Web Annotation
-            </a>
         </div>
     </div>
-    <div class="row">
+
+    <div class="row" style="margin: 30px;">
         <div class="col">
             <h4> Other Linked Open Data Resources </h4>
             <p>
                 <a target="_blank" href="https://lod-cloud.net/">Linked Open Usable Data is a vast expanse</a> of different data classifications with well described taxonomies and ontologies.  If your URI resolves to JSON, preferably JSON-LD <abbr title="?"><a target="_blank" href="https://dashjoin.medium.com/json-schema-schema-org-json-ld-whats-the-difference-e30d7315686a">(what's the difference?)</a></abbr>, then you can provide GeoJSON-LD for that <abbr title="Linked Open Usable Data">LOUD</abbr> entity through using a <a target="_blank" href="https://www.w3.org/TR/annotation-model/">W3C Web Annotation</a>.  This path seeks to finish with a real active <abbr title="Linked Open Usable Data">LOUD</abbr> Geolocating Web Annotation for the resource you provide.
             </p>
             <h5>W3C Web Annotation</h5>
-            <a class="button primary" style="display: inline-block;" href="annotate.html">
-                Create a Geolocating Web Annotation
-            </a>
         </div>
     </div>
 
+    <div class="navLinks">
```

Comment:
Use Chota to do a nav.  That is the framework the site uses to keep everything looking similar.  See https://jenil.github.io/chota/#nav